knowledge_base/Environment-Variables.md

```diff
@@ -211,7 +219,7 @@ Owner: Raymond Zhong.
 
 ## ETH_ALCHEMY_API_KEY
 
-Used in our CI/CD and stored in GitHub repo secrets. If set, the `load-db` package script will replace production Alchemy URLs with their locally supported variants. Only needed if doing work that involves querying Ethereum.
+Must be enabled for testing contests. Also used in our CI/CD and stored in GitHub repo secrets. If set, the `load-db` package script will replace production Alchemy URLs with their locally supported variants. Only needed if doing work that involves querying Ethereum.
```

Comment:
`ETH_ALCHEMY_API_KEY` actually isn't required for contests. I had mistakenly added it to the ticket.